utils/btrfs-progs/Makefile

```diff
@@ -30,7 +30,14 @@ define Package/btrfs-progs
   SUBMENU:=Filesystem
   TITLE:=Btrfs filesystems utilities
   URL:=https://btrfs.wiki.kernel.org/
-  DEPENDS:=+libattr +libuuid +zlib +libblkid +liblzo +libpthread +BTRFS_PROGS_ZSTD:libzstd
+  DEPENDS:=+libattr \
+	  +libuuid \
+	  +zlib \
+	  +libblkid \
+	  +liblzo \
+	  +libpthread \
+	  +BTRFS_PROGS_ZSTD:libzstd \
+	  +kmod-fs-btrfs
```

Comment:
Please do not mix tabs and spaces here. Initial section is indented by two spaces but this is being indented by tab. Replace tab with four spaces to be consistent with two levels of indentation.

Reply:
There is no default, because I have already seen different variants in the openwrt repositories But I have now changed it to your wish,

Reply:
Yeh there is no default I know but I at least try to make it consistent. That means if tabs are used then use tabs and if spaces are used then use spaces.

Of course it is makefile so `Package/Compile` for example always has to be indented by tab no matter what.